test/verify/check-system-services

```diff
@@ -82,7 +82,8 @@ class TestServices(MachineCase):
         self.browser.wait_in_text(self.svc_sel(service), state_new)
 
     def wait_page_load(self):
-        self.browser.wait_not_present(".pf-c-empty-state .pf-c-spinner[aria-valuetext='Loading...']")
+        with self.browser.wait_timeout(180):
```

Comment:
https://github.com/cockpit-project/cockpit/issues/14840

maybe?

Reply:
Could be, but hard to say. It's just consistently happening on F34, but fine on F33 and earlier.